smelter-api/src/output/hls.rs

```diff
@@ -43,6 +43,9 @@ pub enum HlsVideoEncoderOptions {
         /// Encoding bitrate. Default value depends on chosen encoder.
         bitrate: Option<VideoEncoderBitrate>,
 
+        /// (**default=`5000`**) Interval between keyframes, in milliseconds.
+        keyframe_interval_ms: Option<u64>,
```

Comment:
What happens if you set this value to e.g. 5000.234 ? Is this 5000 or validation error?

I don't think we will need sub millisecond precision, so if it is rounding automatically then it is fine, but if not it needs to be a float